pages/docs/configuration/configuration-action.md

```diff
@@ -51,6 +39,10 @@ All target and source data used in actions require `<read-data ... />` or `<writ
 
 For more details, please refer to the [XML reference](configuration-xml-reference.html).
 
+{% version %}
+In older preCICE versions (before v3.0), much more timings and pre-implemented actions were supported. We removed these as their usefulness was limited.
+{% endversion %}
+
```

Comment:
Remove this part?

Reply:
I think such hints are good. Users might still come look here. As long as such hints are short.